src/targets/browser/launcher.ts

```diff
@@ -98,7 +99,7 @@ export async function launch(
     const cp = childProcess.spawn(executablePath, browserArguments.toArray(), {
       detached: true,
       env: env.defined(),
-      cwd,
+      cwd: existsSync(cwd) ? cwd : process.cwd(),
```

Comment:
Since this function is async already, you can use `canAccess` from the `fsUtils` file to avoid blocking the thread :)

Reply:
done!